### PR TITLE
BL-111-fe-as-an-administrator-i-can-approve-or-reject-an-application-from-a-mentee-or-mentor-so-that-i-can-get-approved-users-onboarded

### PR DESCRIPTION
**Pull request by MelodyLayne**
## Description
[Link to Jira Ticket](https://bloomtechlabs.atlassian.net/browse/BL-111)
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Updates to JSX changing the 'onConfirm' handlers to 'onClick' handlers.
Updated Axios calls to point at the correct endpoints for functionality.
No new dependencies

Fixes # (issue)

## Loom Video

https://www.loom.com/share/52a436f14971421b99de5473f002d0c1

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by nathancai97 — DISMISSED** [review]
The updated code with the necessary changes are easy to understand and organized. For your PR, adding a Loom video and a link to the Jira board ticket would be beneficial!

**Review by miguelaledesma — APPROVED** [review]
Great work on this PR. Only thing i would probably do differently is remove console.logs and a Loom before creating the final push to create the PR. Other than that, good fix and great work updating the axios requests and fixing the buttons functionality.

**Comment by MelodyLayne** [comment]
> The updated code with the necessary changes are easy to understand and organized. For your PR, adding a Loom video and a link to the Jira board ticket would be beneficial!

@nathancai97 The Loom will be up today and I completely forgot about adding the link to Jira. Great suggestion and I will add that.

**Comment by MelodyLayne** [comment]
> Great work on this PR. Only thing i would probably do differently is remove console.logs and a Loom before creating the final push to create the PR. Other than that, good fix and great work updating the axios requests and fixing the buttons functionality.

@miguelaledesma thank you for catching that, we'll go clear those console logs and re-push it.

**Review by jettyblue — DISMISSED** [review]
Nice updates here by fixing the buttons function and axios. Looks good, nice work!

**Review by miguelaledesma — APPROVED** [review]
Code looks clean now, no more unnecessary console logs YAY :)  Great job on this fix!

**Review by nathancai97 — APPROVED** [review]
Thank you for updating the PR so quickly! Everything looks ready to be deployed :)

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This is a great fix - but I can't merge it up without a Loom video walking through the changes you made here! 😱 

Please record and post a link to a Loom video walking us through this ASAP and we'll get this merged up!

**Comment by MelodyLayne** [comment]
> This is a great fix - but I can't merge it up without a Loom video walking through the changes you made here! 😱
> 
> Please record and post a link to a Loom video walking us through this ASAP and we'll get this merged up!

@paulstgermain  Loom video link, walkthrough done by the esteemed Christwide Oscar!

**Review by Rlauckern — APPROVED** [review]
Great changes and loom Video. Very organized and professional.

**Review by Rlauckern — DISMISSED** [review]
Looking good. Nice changes from the comments and great organization!

**Review by zdravoj — DISMISSED** [review]
Thanks for connecting this to the correct data pipeline! Nice work.

**Review by miguelaledesma — DISMISSED** [review]
Thank you for removing the console logs, re-reviewing to approve the updated changes

**Review by nathancai97 — APPROVED** [review]
Great job implementing all these changes and updates!

**Review by farhaannishtar — COMMENTED** [review]
Great work hitting the endpoint needed to give the admin the ability to approve / reject the application. 👍

**Review by farhaannishtar — APPROVED** [review]

**Review by jettyblue — APPROVED** [review]
Looks great, nice work and addition of the Loom recording!

**Review by techlando — APPROVED** [review]
well written. good job!
also very helpful comments

File: src/components/pages/PendingApplications/ApplicationModal.js [hunk 1]
```diff
@@ -70,10 +70,17 @@ const ApplicationModal = ({
    * Author: Khaleel Musleh
    * @param {approveApplication} e is for approving an application of a mentor_intake or mentee_intake Boolean from false to approved:true making a PUT call to the backend database server.
    */
-
+  /**
+   * Author: Christwide Oscar
+   * @param {onConfirm} e was not created for the application approve and reject buttons. I changed the functions for the onConfirm to onClick and everything seem to work correctly from the console side.
+   */
   const approveApplication = e => {
+    console.log(currentApplication);
```
Comment (Johnjx) [inline src/components/pages/PendingApplications/ApplicationModal.js:78]: Is this clog still making it through?

**Review by Johnjx — COMMENTED** [review]
Well met addition of code comments to help explain the process. I would highly advise doing a quick walkthrough of the area of the website that concerns this PR, highlighting the elements in question that interact with these functions. For a new entrant into the codebase it is difficult to piece together the items that are being referenced and what the updates are.

**Pull request merged**